### PR TITLE
Reduce console output of lang_update and import_locamotion

### PR DESCRIPTION
**Pull request by flodolo**
Also update commenting style in some other scripts

### CODE REVIEW

File: app/classes/Langchecker/GetTextManager.php [hunk 1]
```diff
@@ -74,16 +74,15 @@ public static function importLocamotion($locale_data, $current_filename, $curren
...
-         *
-         * Use dashes and not underscores for Pootle, e.g. gn-PY (not gn_PY).
+        /*
+            Some locales need to me mapped because our systems and Pootle use
+            different locale codes.
+
+            Array format: Langchecker code => Pootle code
+
+            Use dashes and not underscores for Pootle, e.g. gn-PY instead of gn_PY.
          */
```
Comment (TheoChevalier) [inline app/classes/Langchecker/GetTextManager.php]: In https://github.com/mozfr/transvision/issues/609#issue-131635075 the ending comment section was aligned with the begining, not sure which one we should enforce everywhere (not critical, though :p) I wonder if PHP cs fixer could check that

Comment (flodolo) [inline app/classes/Langchecker/GetTextManager.php]: Uhm, I missed that. I confess that the misalignmed asterisks look a bit weird.

**Comment by TheoChevalier** [comment]
otherwise, lgtm :beers:

File: app/classes/Langchecker/GetTextManager.php (hunk 1 above)
Comment (flodolo) [inline app/classes/Langchecker/GetTextManager.php]: I see php.net has a [page with an example](http://php.net/manual/en/language.basic-syntax.comments.php) like Pascal's style, let's stick to that.

**Pull request merged**